packages/widgetbook_core/lib/src/navigation_tree/models/multi_child_navigation_node_data.dart

```diff
@@ -1,47 +1,23 @@
+import 'package:widgetbook_core/src/navigation_tree/models/navigation_node_data_interface.dart';
 import 'package:widgetbook_core/widgetbook_core.dart';
 
-class MultiChildNavigationNodeData extends NavigationTreeNodeData {
-  MultiChildNavigationNodeData({
+class MultiChildNavigationNodeData extends NavigationNodeDataInterface {
+  const MultiChildNavigationNodeData({
```

Comment:
Since `NavigationNodeDataInterface` is an abstract (non-freezed) class, I think we can make `MultiChildNavigationNodeData` a freezed class.

Reply:
Couldn't do this since `WidgetbookCategory`, `WidgetbookFolder`, `WidgetbookComponent`, and `WidgetbookPackage` extend this class